utils/update_checkout/update-checkout-config.json

```diff
@@ -88,7 +84,22 @@
         "llvm-project": {
             "remote": { "id": "apple/llvm-project" } },
         "wasi-libc": {
-            "remote": { "id": "WebAssembly/wasi-libc" } }
+            "remote": { "id": "WebAssembly/wasi-libc" } },
+        "curl": {
+            "remote": { "id": "curl/curl" }
+        },
+        "icu": {
+            "remote": { "id": "unicode-org/icu" }
+        },
+        "libxml2": {
+            "remote": { "id": "gnome/libxml2" }
+        },
+        "sqlite": {
+            "remote": { "id": "sqlite/sqlite" }
```

Comment:
How exactly are we planning on using this checkout? The git mirror does not have the amalgamation so we would need to install tcl on CI and on machines that folks want to use to build sqlite. Are we sure that we want to recommend this route?

Reply:
Well, @shahmishal requested that we have a means for not having to clone the dependencies. I suppose that the alternative is having Jenkins fetch the amalgamation for us.  I would _strongly_ prefer having the amalgamation.

Reply:
I mean, given that the sqlite folks explicitly say that distributions should use the amalgamation (and the split amalgamation for debug builds), I'm very much inclined to do that. Without the amalgamation, wrapping the sqlite build for build-script/CMake gets a fair bit trickier too.